python/setup.py

```diff
@@ -146,6 +146,11 @@ def find_version(*filepath):
     "six >= 1.0.0",
     "flatbuffers",
     "faulthandler;python_version<'3.3'",
+    # simplejson and psutil are required by the Reporter.
+    "simplejson",
+    "psutil",
```

Comment:
We can't make `psutil` a hard requirement because installation of `psutil` is too flaky and leads to a lot of errors. I removed it in https://github.com/ray-project/ray/pull/2892 (see also https://github.com/ray-project/ray/issues/2806).

We could have a separate `pip install ray[webui]` or something like that.

Reply:
I don't think we should get rid of `psutil` from `reporter.py`. Instead, I think we should just not start the UI if `psutil` is not installed.

Reply:
I'd prefer this approach. We could just dump an error message along the lines of 'not starting as psutil not installed'.

Reply:
`reporter` bails out if psutil not present.